client/src/components/Site.vue

```diff
@@ -16,11 +16,21 @@
       </p>
     </div>
     <div>
+      <p class="level-heading heading"> Latest Closure Information </p>
```

Comment:
This is good, but it wastes a lot of space on the screen (an entire graph's worth). Can we try moving this somewhere? Perhaps on the same line as the "OK to open" title. Or somewhere near/on that graph.